tests/nodes.yaml

```diff
@@ -0,0 +1,232 @@
+apiVersion: v1
```

Comment:
I think you can and should make this files _much_ smaller. As a start remove everything under `status` - it's read only anyway